packages/sources/src/components/description/description.args.ts

```diff
@@ -52,6 +47,25 @@ export function descriptionArgsMapper(a: DescriptionArgs): Description {
     content: a.content,
     id: a.id,
     open: a.open,
-    term: a.term
+    term: a.term,
+    type: a.type
   };
 }
+
+export interface DescriptionExampleArgs {
+  openTerm: boolean;
```

Comment:
Waarom zou je deze als losse argumenten willen? Een description is toch altijd een term OF een note, dus is alleen een "open" (of: "openDescription") dan niet goed genoeg?

Reply:
Aha ik snap het al, je doet dit om juist de combinatie te kunnen laten zien, en dan wel beide vormen onafhankelijk van elkaar te kunnen bedienen. Creatief.
![image](https://user-images.githubusercontent.com/2689559/136770291-ac783d64-bc09-4281-9192-e7bf9d0a1a20.png)